app/scripts/directives/logViewer.js

```diff
@@ -45,7 +47,7 @@ angular.module('openshiftConsole')
         scope: {
           followAffixTop: '=?',
           followAffixBottom: '=?',
-          resource: '@',
+          object: '=',
```

Comment:
Is object clear enough?  Should it be consistent with `APIService`, like `apiObject` `api-object="whatever"` or `qualifiedResource`?